compiler/oxidate/src/compiler.rs

```diff
@@ -333,4 +348,36 @@ mod tests {
         ];
         assert_eq!(res, exp);
     }
+
+    #[test]
+    fn test_compile_assign() {
+        let res = exp_compile_str("let x = 2; x = 3;");
+        let exp = vec![
+            LDC(Int(2)),
+            ASSIGN("x".to_string()),
+            LDC(Unit),
+            POP,
+            LDC(Int(3)),
+            ASSIGN("x".to_string()),
```

Comment:
Btw dude, you can use some of lowercase methods that takes into Into<Value>, hopefully it makes things easier

```rst
ByteCode::ldc(1);
ByteCode::ldc("very convenient");
```

Reply:
Thanks a lot!